### PR TITLE
Implement Dashboard - Add User Succesfully

### PR DESCRIPTION
**Pull request by DKnehi**

### CODE REVIEW

File: javascript-practice/src/js/app.js [hunk 1]
```diff
@@ -16,7 +16,8 @@ export default class App {
       view.bindFormLogin(controller.handleFormLogin);
     } else if (path.includes(PATHS.DASHBOARD)) {
       const view = new DashboardView();
-      // const controller = new DashboardController(model, view);
+      const controller = new DashboardController(model, view);
```
Comment (phuocduong-agilityio) [inline javascript-practice/src/js/app.js:19]: Why pass view into controller then use view to init app here?

Comment (phuocduong-agilityio) [inline javascript-practice/src/js/app.js]: DashboardController don't have any parameter. Why you put view and model into this?

File: javascript-practice/src/js/controllers/dashboardController.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import DashboardView from '../views/dashboardView';
+
+export default class DashboardController {
+  constructor() {
+    this.view = new DashboardView();
+    this.model = new UserModel();
+    this.service = new UserService();
+  }
+
+  addUser = async (
```
Comment (phuocduong-agilityio) [inline javascript-practice/src/js/controllers/dashboardController.js:20]: A good function should NOT have more than 3 parameters, pls consider another way

File: javascript-practice/src/js/views/dashboardView.js [hunk 3]
```diff
@@ -49,6 +60,38 @@ export default class DashboardView {
     });
   };
 
+  bindRoleSelection = (event) => {
+    const valueSelectRole = event.target.value;
+    console.log(valueSelectRole);
```
Comment (phuocduong-agilityio) [inline javascript-practice/src/js/views/dashboardView.js]: Remove console.log for debugging on local

File: javascript-practice/src/js/controllers/dashboardController.js (hunk 2 above)
Comment (DKnehi) [reply javascript-practice/src/js/controllers/dashboardController.js:20]: I'll update later

File: javascript-practice/src/js/app.js (hunk 1 above)
Comment (DKnehi) [reply javascript-practice/src/js/app.js]: I'll update later

Comment (phuocduong-agilityio) [reply javascript-practice/src/js/app.js]: Not accept this, it didn't take much time, just need move it to controller

File: javascript-practice/src/js/controllers/dashboardController.js (hunk 2 above)
Comment (phuocduong-agilityio) [reply javascript-practice/src/js/controllers/dashboardController.js:20]: Got it

File: javascript-practice/src/js/app.js [hunk 4]
```diff
@@ -12,11 +12,11 @@ export default class App {
 
     if (path.includes(PATHS.LOGIN)) {
       const view = new UserView();
-      const controller = new UserController(model, view);
+      const controller = new UserController();
       view.bindFormLogin(controller.handleFormLogin);
     } else if (path.includes(PATHS.DASHBOARD)) {
       const view = new DashboardView();
-      const controller = new DashboardController(model, view);
+      const controller = new DashboardController();
```
Comment (phuocduong-agilityio) [inline javascript-practice/src/js/app.js]: All view initials should be inside controller. Controller is a place that we control the app follow. For current implement, we can not control app follow. Ping me if you can not catch an idea

**Pull request merged**